format/index.md

```diff
@@ -34,20 +34,21 @@ the header `Content-Type: application/vnd.api+json`.
 Clients **MUST** ignore all parameters for the `application/vnd.api+json`
 media type received in the `Content-Type` header of response documents.
 
```

Comment:
I thought that too for a while. But I decided to leave it because we might, in the future, want to let the server to automatically opt the client in to some parameters (e.g. `profile`, which just adds _extra_ semantics) and, if we do that, we don't want old clients to blow up.